docs/resources/Guild.md

```diff
@@ -15,13 +15,12 @@ Guilds in Discord represent an isolated collection of users and channels, and ar
 | owner?                        | boolean                                                                             | true if [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) is the owner of the guild                                                |
 | owner_id                      | snowflake                                                                           | id of owner                                                                                                                               |
-| permissions?                  | integer                                                                             | legacy total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                   |
-| permissions_new?              | string                                                                              | total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                          |
+| permissions?                  | string                                                                             | legacy total permissions for [the user](#DOCS_RESOURCES_USER/get-current-user-guilds) in the guild (excludes overrides)                   |
 | region                        | string                                                                              | [voice region](#DOCS_RESOURCES_VOICE/voice-region-object) id for the guild                                                                |
 | afk_channel_id                | ?snowflake                                                                          | id of afk channel                                                                                                                         |
 | afk_timeout                   | integer                                                                             | afk timeout in seconds                                                                                                                    |
-| embed_enabled?                | boolean                                                                             | true if the server widget is enabled (deprecated, replaced with `widget_enabled`)                                                         |
-| embed_channel_id?             | ?snowflake                                                                          | the channel id that the widget will generate an invite to, or `null` if set to no invite (deprecated, replaced with `widget_channel_id`)  |
+| widget_enabled?                | boolean                                                                             | true if the server widget is enabled (deprecated, replaced with `widget_enabled`)                                                         |
```

Comment:
these are now in this list twice

Reply:
also on lines 172-173

Reply:
perhaps it would be worth sorting these tables to make them easier to read?